### PR TITLE
Create batjoker

### PR DESCRIPTION
**Pull request by braydonbibe**

### CODE REVIEW

**Comment by braydonbibe** [comment]
costumes:
- color: '&5'
  name: §fJo§2k§5er
  sig: AwQXimwxx8rTzOYRYqOVy03JYdNhKcjcc9TpQj+CybBC1Nf/mKV5kPk5AMpxPxA+DPvf+bxt4vdC5tUn+mQ8hvRymcidxr8ST4Ztdy6EYsmo3seAzONko3tSq4v5ccoSwVTbnj/6Hp5XYJMkBEik7Szm7rnz55dvl/MHTnc6TQ2dto9siCUu1GX35AqsiQHBBQ6XlKPAdegEWL8783aSs542Inad0uvq2b3nG3cnP1PXES6qCLxPKciKf+lwasIIqFyU+3qXIDxw8CsXpISGhvj/JQv+Snn5sjvWK/vSlxm3W1FRAZJSYdHAK0HkF7gOnHuoKYQ7eEe6LVmelQ71h6t3PpvGBXaV6wnetnCN9zv86SbOhiotOT+7FW0KVext6uY7XfLYPQj+ggLsIhKzFPw6RiCH7Xj2J8b3Gwo9spUGcFvfzLUMIv37Nob1b1QQ2jU2MZFVvCJSh7TOrzdKFrge1rG49CULt8wtwPI6PfOEVi+iXOpGVQinu+ANBDuMbR9dI5bkuFQ42vn01e0bIgbPJb12GjBjiWRmaZEODlzt31/EgOJHzcaRi5dLaUz9TmLn4XeVkczqy1SVwNCAfn8G8b5HlPreGGCDEL2NaRE1BfHzQ1fbvhJ/NI41yuimbfOxidw4cnQuqmq42Vjxqry46lQK9b7/Nsx21k7MtYY=
  val: eyJ0aW1lc3RhbXAiOjE0NzQ4MjU4NjQ2NjcsInByb2ZpbGVJZCI6Ijg3ZmIzYzZjNWZmNjRiZjZiMzI2OTkyYzYxMzdlYTEyIiwicHJvZmlsZU5hbWUiOiJUZWt0cm9uaWMiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlL2RlZjZhOGYwNjU4MDQwZTk0NmI4NjViYmU1ODdjYzVkZjYwMzVkM2MwOWQ1NDNmYjI5MzZiZjI3YWJkMzE4In19fQ==
buffs:
- /give <plr> minecraft:stick 1 16:2 name:&7Crow_Bar
- /shot give <plr> pistol
- /shot give <plr> jokbomb
- /give <plr> minecraft:elytra 1 70:1 34:3 name:&7B&ea&0t&7m&8a&7n's_&8Glide_Wings
- /give <plr> minecraft:snowball 10 name:&8jokarang
- /give <plr> minecraft:ender_pearl 5 name:&7B&ea&0t&7m&8a&7n's_&8jok_o_Hook
class: joker
haste: 3
jump: 4
maxHearts: 35
op: false
powers:
- /buff
- /jump <0-3>
- /run <0-2>
- /haste <0-2>
sig: AwQXimwxx8rTzOYRYqOVy03JYdNhKcjcc9TpQj+CybBC1Nf/mKV5kPk5AMpxPxA+DPvf+bxt4vdC5tUn+mQ8hvRymcidxr8ST4Ztdy6EYsmo3seAzONko3tSq4v5ccoSwVTbnj/6Hp5XYJMkBEik7Szm7rnz55dvl/MHTnc6TQ2dto9siCUu1GX35AqsiQHBBQ6XlKPAdegEWL8783aSs542Inad0uvq2b3nG3cnP1PXES6qCLxPKciKf+lwasIIqFyU+3qXIDxw8CsXpISGhvj/JQv+Snn5sjvWK/vSlxm3W1FRAZJSYdHAK0HkF7gOnHuoKYQ7eEe6LVmelQ71h6t3PpvGBXaV6wnetnCN9zv86SbOhiotOT+7FW0KVext6uY7XfLYPQj+ggLsIhKzFPw6RiCH7Xj2J8b3Gwo9spUGcFvfzLUMIv37Nob1b1QQ2jU2MZFVvCJSh7TOrzdKFrge1rG49CULt8wtwPI6PfOEVi+iXOpGVQinu+ANBDuMbR9dI5bkuFQ42vn01e0bIgbPJb12GjBjiWRmaZEODlzt31/EgOJHzcaRi5dLaUz9TmLn4XeVkczqy1SVwNCAfn8G8b5HlPreGGCDEL2NaRE1BfHzQ1fbvhJ/NI41yuimbfOxidw4cnQuqmq42Vjxqry46lQK9b7/Nsx21k7MtYY=
speech: I KILL THE BATMAN!
trail: ''
val: eyJ0aW1lc3RhbXAiOjE0NzQ4MjU4NjQ2NjcsInByb2ZpbGVJZCI6Ijg3ZmIzYzZjNWZmNjRiZjZiMzI2OTkyYzYxMzdlYTEyIiwicHJvZmlsZU5hbWUiOiJUZWt0cm9uaWMiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlL2RlZjZhOGYwNjU4MDQwZTk0NmI4NjViYmU1ODdjYzVkZjYwMzVkM2MwOWQ1NDNmYjI5MzZiZjI3YWJkMzE4In19fQ==
altskins https://i.imgur.com/KUDNVWr.png

File: CommandHelper/LocalPackages/commands/chars/batjoker [hunk 1]
```diff
@@ -0,0 +1,27 @@
+costumes:
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/batjoker]: This is really good!

The only issue I see is that it the filename up there should be

CommandHelper/LocalPackages/commands/chars/batjoker.yml
not
CommandHelper/LocalPackages/commands/chars/batjoker

notice it should have .yml

thanks!

**Comment by malonnnn** [comment]
dont forget to include proofs of the powers!

File: CommandHelper/LocalPackages/commands/chars/batjoker (hunk 1 above)
Comment (braydonbibe) [reply CommandHelper/LocalPackages/commands/chars/batjoker]: ok im on it

**Pull request closed without merging**